### PR TITLE
Change product package usage to .zip files

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: dev/group_vars/is-analytics.yml [hunk 1]
```diff
@@ -16,15 +16,18 @@
...
 product_name: wso2is-analytics
 product_version: 5.8.0
-product_path: /opt/wso2
-product_archive_deb: wso2is-analytics-linux-installer-x64-5.8.0.deb
-product_archive_rpm: wso2is-analytics-linux-installer-x64-5.8.0.rpm
+product_path: /usr/local/wso2
 product_package_location: files
-jdbc_driver: mysql-connector-java-5.1.45-bin.jar
+jdbc_driver: mysql-connector-java-5.1.47-bin.jar
+
```
Comment (VimukthiPerera) [inline dev/group_vars/is-analytics.yml]: Shall we remove the extra line here?

File: dev/group_vars/is.yml [hunk 2]
```diff
@@ -16,15 +16,18 @@
...
 product_name: wso2is
 product_version: 5.8.0
-product_path: /opt/wso2
-product_archive_deb: wso2is-linux-installer-x64-5.8.0.deb
-product_archive_rpm: wso2is-linux-installer-x64-5.8.0.rpm
+product_path: /usr/local/wso2
 product_package_location: files
-jdbc_driver: mysql-connector-java-5.1.45-bin.jar
+jdbc_driver: mysql-connector-java-5.1.47-bin.jar
+
```
Comment (VimukthiPerera) [inline dev/group_vars/is.yml]: Extra line.

File: dev/group_vars/is-analytics.yml (hunk 1 above)
Comment (Savidude) [reply dev/group_vars/is-analytics.yml]: This was done to separate JDK related variables from the rest. Shall I add a comment indicating that the variables after the space are related to JDK distributions?

Comment (Savidude) [reply dev/group_vars/is-analytics.yml]: Fixed https://github.com/wso2/ansible-is/pull/17/commits/b99772a015e08081ff91fd679fec49857ae63e14

File: dev/group_vars/is.yml (hunk 2 above)
Comment (Savidude) [reply dev/group_vars/is.yml]: Fixed https://github.com/wso2/ansible-is/pull/17/commits/b99772a015e08081ff91fd679fec49857ae63e14

**Pull request merged**